src/modules/navigator/mission_block.cpp

```diff
@@ -279,7 +279,8 @@ MissionBlock::is_mission_item_reached()
 
 			// We use the acceptance radius of the mission item if it has been set (not NAN)
 			// but only for multicopter.
```

Comment:
`// but only for multicopter.` can be removed here.

Reply:
Thanks, I missed that.